src/features/user/Account/components/PaymentRecord.tsx

```diff
@@ -0,0 +1,150 @@
+import { Modal } from '@material-ui/core';
+import React, { ReactElement } from 'react';
+import BackButton from '../../../../../out/assets/images/icons/BackButton';
```

Comment:
Do no import stuff from the `out` folder which is generated while deploying. Using `public` instead would be right. Using `out` shows an error if running this locally with `npm run dev`.